maps/scrape.py

```diff
@@ -16,25 +17,34 @@ def scrape_calls(date1: datetime, date2: datetime):
     return j
 
 
-def insert_data(calls):
-    with sqlite3.connect(DB_PATH) as conn:
-        c = conn.cursor()
-        prepared = []
-        for call in calls:
-            dt = datetime.strptime(call["DispatchTime"] + " " + call["DispatchTime2"], "%m-%d-%Y %H:%M:%S")
-            values = (dt, call["lat"], call["lon"], call["City"], call["CallType"], call["Address"])
-            prepared.append(values)
-        c.executemany('''INSERT OR REPLACE INTO calls (datetime, lat, lon, city, call_type, address)
-        VALUES (?,?,?,?,?,?)''', prepared)
-
-    conn.commit()
-
-
-def fetch_data(days_ago=1):
+def insert_data(call_data, v=0):
```

Comment:
What is V?

Reply:
Some more comments here would help, as I'm kinda at a loss for what's happening.

Reply:
V is verbosity; if you want no more logging from the scraper run it with v=0